production/technical_userguide.md

```diff
@@ -0,0 +1,518 @@
+---
+layout: home
+title:  "Advanced Production User Guide"
+description: "A technical guide to getting set up in BCDA and retrieving beneficiary data."
+date:   2019-09-06 09:21:12 -0500
+landing-page: live
+gradient: "blueberry-lime-background"
+subnav-link-gradient: "blueberry-lime-link"
+nav_link: production
+permalink: /production/technical-user-guide/
+id: prod-technical-user-guide
+sections:
+  - User Guide
+  - Authentication and Authorization
+  - Encryption
+  - Environment
+  - Examples
+---
+
+## User Guide
+
+The Beneficiary Claims Data API (BCDA) enables Accountable Care Organizations (ACOs) to retrieve claims data for their Medicare beneficiaries. This includes claims data for instances in which beneficiaries receive care outside of the ACO, allowing a full picture of patient care.
+
+This API follows the workflow outlined by the [FHIR Bulk Data Export Proposal](https://github.com/HL7/bulk-data/blob/master/spec/export/index.md){:target="_blank"} using the [HL7 FHIR Standard](https://www.hl7.org/fhir/){:target="_blank"}. Claims data is provided as FHIR resources in [NDJSON](http://ndjson.org){:target="_blank"} format.
+
+This guide serves as a starting point for users to begin working with the API. [Comprehensive Swagger documentation about all BCDA endpoints is also available.](https://api.bcda.cms.gov/api/v1/swagger){:target="_blank"}
+
+**Note:** if this documentation is a little too in-depth, you may want to start with our [guide to getting started in production](/production/user-guide/) for an overview of APIs and a gentler walk-through.
+
+**Note:** If you have not yet received production credentials, you will not be able to access production endpoints and data. To put your ACO in the queue for access, please send an email to [bcapi@cms.hhs.gov](mailto:bcapi@cms.hhs.gov) with your name and your ACO’s name, ID number, and track. ACOs will be onboarded to production in the order in which requests are received.. While you wait, you can [get familiar with the API in the sandbox environment](/sandbox/user-guide/), review the [data structure](/data-guide/) and join the [BCDA Google Group](https://groups.google.com/forum/#!forum/bc-api){:target="_blank"}, to have your questions answered.
```

Comment:
A wild period appears! "in which requests are received" should only be followed by one period.